admin_guide/sdn_troubleshooting.adoc

```diff
@@ -398,7 +398,14 @@ default via 192.168.122.1 dev ens3
 
 You should see the 10.128.x.x lines (assuming you have your pod network set to the default range in your configuration). If you do not, check the {product-title} logs (see the <<reading-the-logs>> section)
 
-=== Is the Open vSwitch configured correctly?
+=== Is the Open vSwitch (OVS) configured correctly?
+
+You must install the `openvswitch` package on the host to run the `ovs-vsctl` and `ovs-ofctl` commands.
+To install the `openvswitch` package, enter the following command:
+
+----
+# yum install openvswitch
+----
```

Comment:
hmm, for 3.11 maybe host do not have `yum` command.  So I suggest run this ovs command in the container: eg
`oc exec -n openshift-sdn ovs-45dzz -- ovs-vsctl list-br`

Reply:
@zhaozhanqi, how do I find the names of the SDN pods? Thanks!

Reply:
```
oc get pod -n openshift-sdn -l app=ovs
NAME        READY     STATUS    RESTARTS   AGE
ovs-45dzz   1/1       Running   0          17h
ovs-8njfh   1/1       Running   0          17h
ovs-zw5tz   1/1       Running   0          17h
oc get pod -n openshift-sdn -l app=sdn
NAME        READY     STATUS    RESTARTS   AGE
sdn-jssrh   1/1       Running   0          17h
sdn-mqbhf   1/1       Running   0          17h
sdn-qzvks   1/1       Running   0          17h
```

Reply:
Thanks!